features.json

```diff
@@ -50,6 +50,11 @@
 			"url": "https://github.com/WebAssembly/reference-types/blob/master/proposals/reference-types/Overview.md",
 			"phase": 4
 		},
+		"relaxedSimd": {
+			"description": "Extensions to fixed-width SIMD with local nondeterminism",
```

Comment:
Hmm, one more thing: `description` is actually the human-readable name that goes into the first column on https://webassembly.org/roadmap/.

I'm generally trying to keep it short where possible, otherwise it will shift all the other columns further out of the viewport.

How about renaming this to just "Relaxed SIMD"?